source/mqtt_agent.c

```diff
@@ -657,7 +657,6 @@ static void mqttEventCallback( MQTTContext_t * pMqttContext,
                  * PINGRESP with the use of MQTT_ProcessLoop API function. */
                 LogWarn( ( "PINGRESP should not be handled by the application "
                            "callback when using MQTT_ProcessLoop.\n" ) );
-                break;
```

Comment:
Why is this removed?

Reply:
To cover the default case as it was not being covered otherwise by any other unit test case.

Reply:
Will this will result in contradicting messages - One, saying that `PINGRESP` is received and should not be handled and Second, saying that unknown packet type received?

Reply:
Yeah, will address it.